src/Github-api.js

```diff
@@ -0,0 +1,42 @@
+const request = require("request-promise");
+
+const host = 'https://api.github.com';
+
+class Github {
+    constructor() {
+        this.request = request.defaults({
+            json: true,
+            headers: {
+                "Content-Type": "application/json",
+                "User-Agent": "Agent"
+            },
+        });
+    }
+
+    getUser(username) {
+        const path = `${host}/users/${username}`;
+
+        return this.request.get({
+            url: path            
+        });
+    }  
+
+    getUsers() {
+        const path = `${host}/users/${username}`;
+
+        return this.request.get({
+            url:path
+        });
+    } 
+    
+    updateUser(userId, body) {
```

Comment:
The update endpoint in GitHub works differently: https://docs.github.com/en/rest/users/users#update-the-authenticated-user
It doesn't take the path parameter, but rather updates the user who's authentication token is passed in the header. I.e. if you want to make it to work, you'd have to delete `userId` and only require a body.